### PR TITLE
build(upgrade): upgrade to version 11 of Angular

### PR DESCRIPTION
**Pull request by DuncanFaulkner**
Upgraded application to use Angular 11 from version Angular 10 in relation to ticket #215

### CODE REVIEW

File: .browserslistrc [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 # You can see what browsers were selected by your queries by running:
 #   npx browserslist
 
-> 0.5%
-last 2 versions
+last 1 Chrome version
+last 1 Firefox version
+last 2 Edge major versions
+last 2 Safari major versions
+last 2 iOS major versions
```
Comment (crisbeto) [inline .browserslistrc:15]: Why was this file changed? Is it part of `ng update`?

File: package.json [hunk 2]
```diff
@@ -29,28 +29,24 @@
     "url": "https://github.com/crisbeto/angular-svg-round-progressbar/issues"
   },
   "homepage": "https://github.com/crisbeto/angular-svg-round-progressbar",
-  "peerDependencies": {
-    "@angular/core": "^10.0.0",
-    "@angular/common": "^10.0.0"
-  },
+  "peerDependencies": {},
```
Comment (crisbeto) [inline package.json:32]: Why were the `peerDependencies` removed?

File: .browserslistrc (hunk 1 above)
Comment (DuncanFaulkner) [reply .browserslistrc:15]: I updated this file after the upgrade to ng 11 as it was issuing warnings relating to IE 9/10 no longer being supported. I just copied this file from a new ng 11 project. happy to roll this one back

File: package.json (hunk 2 above)
Comment (DuncanFaulkner) [reply package.json:32]: I will have a look into this. I can only assume the upgrade removed them

File: .browserslistrc (hunk 1 above)
Comment (crisbeto) [reply .browserslistrc:15]: I think it should be reverted since the module does still work down to IE9. I assume that Angular removed it, because you'd need extra polyfills to support IE.

File: package.json (hunk 2 above)
Comment (crisbeto) [reply package.json:32]: I think it should be reverted. I've based this on how it's being done by Angular Material.

File: .browserslistrc (hunk 1 above)
Comment (DuncanFaulkner) [reply .browserslistrc]: I have reverted this file

File: package.json (hunk 2 above)
Comment (DuncanFaulkner) [reply package.json]: I have added back missing peer dependencies

**Review by crisbeto — APPROVED** [review]
LGTM
